yaml/_error.ts

```diff
@@ -14,7 +14,7 @@ export class YAMLError extends Error {
     this.name = this.constructor.name;
   }
 
-  public override toString(_compact: boolean): string {
+  override toString(_compact: boolean): string {
```

Comment:
Nit: let's remove this variable since it's not used.
```suggestion
  override toString(): string {
```